### DISCUSSION TITLE
Recommendations for achieving single-origin with Backend/API

### DISCUSSION THREAD

**Discussion by kennym** [vercel/community#743] [Help] [answered]
We have our frontend app (Next) and a Django backend. Is there a way to achieve single origin with frontend <-> backend without CORS?

**Comment by amyegan** [comment, marked as answer]
Hi @kennym. You can rewrite URLs in `next.config.js` or `vercel.json`.
- In Next config: https://nextjs.org/docs/api-reference/next.config.js/rewrites
- in Vercel Project config: https://vercel.com/docs/project-configuration#project-configuration/rewrites
